examples/cdn/index.html

```diff
@@ -2,7 +2,7 @@
 <meta charset="utf-8"/>
 <html>
   <head>
-    <script src="https://cdn-webgl.wrld3d.com/wrldjs/dist/v1.0.0/wrld.js"></script>
+    <script src="https://unpkg.com/wrld.js@1.x.x"></script>

```

Comment:
`1.x.x` is a valid string, however it won't work until we publish 1.0.0 to the latest channel. I intentionally kept it this way, so that it would be ready for release.